terraform/environment/example.tfvars

```diff
@@ -0,0 +1,3 @@
+aws_profile = ""
+web_domain_name = ""
+aws_region = ""
```

Comment:
In the example here I think we should fill in the default values that exist in the variables.tf (aws_profile and aws_region). I'm starting to prefer this over excluding them completely, that way the user sees and replaces the value when needed.

Reply:
I was thinking of doing that.